lib/badge.js

```diff
@@ -0,0 +1,33 @@
+export const CLEAR_BADGE_ACTION = 'CLEAR_BADGE'
+
+export const dispatchClearBadgeAction = () => navigator.serviceWorker?.controller?.postMessage({ action: CLEAR_BADGE_ACTION })
```

Comment:
I would rename this function to `clearNotifications` and the action to `CLEAR_NOTIFICATIONS` since it's more about clearing the notifications. It only calls the `clearAppBadge` function defined below in this file. The name confused me since there were two functions which seemed to clear the badge.

Reply:
Yep, that makes sense. Another case of evolving functionality without renaming. I'll clean it up :)